services/catalog/requirements/_base.in

```diff
@@ -21,12 +21,14 @@ fastapi[all]
 pydantic[dotenv]
 
 # database
-aiopg[sa]
+asyncpg
+sqlalchemy[asyncio]
```

Comment:
two things, we already introduce this "extra"
```
sqlalchemy[postgresql_psycopg2binary]
```
should we always extra with both?
```
sqlalchemy[asyncio,postgresql_psycopg2binary]
```

Reply:
no I think once we can fully migrate, the psycopg2binary becomes obsolete. maybe just for testing where we do have a sync engine as well